### DISCUSSION TITLE
Request for PowerShell Example of LiveCharts MaterialCards

### DISCUSSION THREAD

**Discussion by H1-N1** [Live-Charts/LiveCharts2#1050] [Ideas]
Hello,

I'm a software developer interested in using LiveCharts MaterialCards to create interactive charts in my PowerShell scripts. However, I am having difficulty finding any PowerShell examples that showcase this feature.

I was wondering if anyone in the community could provide me with a working example or point me in the right direction. Specifically, I am interested in learning how to customize the MaterialCards appearance and how to integrate it with other LiveCharts features.

Any guidance or resources would be greatly appreciated. Thank you in advance for your help.

Best regards,
joe

**Comment by beto-rodriguez** [comment]
Hello and thanks for the Issue, the material cards sample belongs to LiveCharts 0, At this point of the project there are no design samples yet, but there will be uploaded in a future version, and a sample using PowerShell seems also interesting! I will definitely do that.

I will close this for now, we can continue this as a discussion.
